includes/HashingService.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace DataAccounting;
+
+class HashingService {
+
+	public function __construct(
+		private string $domainId
+	) {
+	}
+
+	public function calculateMetadataHash( $timestamp, $previousVerificationHash = "" ): string {
```

Comment:
What if we want to export these functions as a separate library? Should it be expected that the library user would use this class?

Reply:
Do you want to create a PHP library with these functions?

Reply:
Yes, if this allows us to re-use the code in e.g. future projects like building a word-press extension and or if it increases maintainability this would be very helpful.

Reply:
So you'd basically want to decouple framework and application bound code from your domain code, allowing you to reuse all domain code elsewhere. Something like Ports and Adapters (https://github.com/inblockio/DataAccounting/pull/153#issuecomment-971515422) does exactly that. And this PR is an example of how you move into that direction.

Reply:
Perfect! Thank you very much